src/catkin_pkg/condition.py

```diff
@@ -58,6 +58,20 @@ def _evaluate(parse_results, context):
         return parse_results
 
     # recursion
+    if len(parse_results) != 3:
+        arg = parse_results[0]
+        condition_len = len(parse_results)
+        parse_list = []
+
+        # After the loop ends, a final evaluation is done
+        for i in range(int(condition_len/2-1)):
```

Comment:
Nitpick: please move the `- 1` out of the `int()` call.

(Might be obsolete with the next comment.)